src/pattern-library/examples/select-right.tsx

```diff
@@ -3,11 +3,11 @@ import { useId, useState } from 'preact/hooks';
 import { Select } from '../..';
 
 const items = [
-  { id: '1', name: 'All students' },
-  { id: '2', name: 'Albert Banana' },
-  { id: '3', name: 'Bernard California' },
-  { id: '4', name: 'Cecelia Davenport' },
-  { id: '5', name: 'Doris Evanescence' },
+  { id: '1', name: 'All students (this item has a very long text)' },
```

Comment:
```suggestion
  { id: '1', name: 'All students (this item has very long text)' },
```

In this context _text_ is an uncountable noun, so no "a".